lib/index.js

```diff
@@ -418,5 +450,6 @@ exports.metadataFunctions = {
 	'general': exports.parseGeneral,
 	'highwirePress': exports.parseHighwirePress,
 	'schemaOrg': exports.parseSchemaOrgMicrodata,
-	'openGraph': exports.parseOpenGraph
+	'openGraph': exports.parseOpenGraph,
+	'eprints': exports.parseEprints
```

Comment:
I kind of suspected, but then thought it didn't matter because openGraph was already after schemaOrg... I'll fix that.